I'm not sure if this means the tests are passing or not... @roryabraham can you please advise?

```
➜  App git:(Rory-ShellCheck) npm run shellcheck

> new.expensify@1.2.36-2 shellcheck
> ./scripts/shellCheck.sh

👀 Linting the following shell scripts using ShellCheck: ./.husky/_/husky.sh
./tests/unit/getPullRequestsMergedBetweenTest.sh
./scripts/shellCheck.sh
./scripts/build-desktop.sh
./scripts/set-pusher-suffix.sh
./scripts/shellUtils.sh
./.github/scripts/validateActionsAndWorkflows.sh
./.github/scripts/verifyPodfile.sh
./.github/scripts/buildActions.sh
./.github/scripts/verifyActions.sh


In ./.husky/_/husky.sh line 9:
  readonly hook_name="$(basename -- "$0")"
           ^-------^ SC2155 (warning): Declare and assign separately to avoid masking return values.


In ./.husky/_/husky.sh line 19:
    . ~/.huskyrc
      ^--------^ SC1090 (warning): ShellCheck can't follow non-constant source. Use a directive to specify location.

For more information:
  https://www.shellcheck.net/wiki/SC1090 -- ShellCheck can't follow non-const...
  https://www.shellcheck.net/wiki/SC2155 -- Declare and assign separately to ...
➜  App git:(Rory-ShellCheck)
```

This happens for me while executing step 1

--- a/scripts/shellCheck.sh
+++ b/scripts/shellCheck.sh
@@ -15,7 +15,8 @@ declare -r DIRECTORIES_TO_IGNORE=(
 )
 
 # This lists all shell scripts in this repo except those in directories we want to ignore
-SHELL_SCRIPTS=$(find . -type d \( -path "$(join_by_string " -o -path " "${DIRECTORIES_TO_IGNORE[@]}")" \) -prune -o -name '*.sh' -print)
+read -ra IGNORE_DIRS < <(join_by_string ' -o -path ' "${DIRECTORIES_TO_IGNORE[@]}")
+SHELL_SCRIPTS=$(find . -type d \( -path "${IGNORE_DIRS[@]}" \) -prune -o -name '*.sh' -print)
 info "👀 Linting the following shell scripts using ShellCheck: $SHELL_SCRIPTS"
 info
 

--- a/scripts/shellCheck.sh
+++ b/scripts/shellCheck.sh
@@ -8,7 +8,7 @@ cd "$ROOT_DIR" || exit 1
 source scripts/shellUtils.sh
 
 # This lists all shell scripts in this repo except those in directories we want to ignore
-SHELL_SCRIPTS=$(find . -type d \( -path ./node_modules -o -path ./vendor -o -path ./ios/Pods \) -prune -o -name '*.sh' -print)
+SHELL_SCRIPTS=$(find . -type d \( -path ./node_modules -o -path ./vendor -o -path ./ios/Pods -o -path ./.husky \) -prune -o -name '*.sh' -print)
 info "👀 Linting the following shell scripts using ShellCheck: $SHELL_SCRIPTS"
 info
 

--- a/scripts/shellCheck.sh
+++ b/scripts/shellCheck.sh
@@ -7,8 +7,15 @@ cd "$ROOT_DIR" || exit 1
 
 source scripts/shellUtils.sh
 
+declare -r DIRECTORIES_TO_IGNORE=(
+  './node_modules'
+  './vendor'
+  './ios/Pods'
+  './.husky'
+)
+
 # This lists all shell scripts in this repo except those in directories we want to ignore
-SHELL_SCRIPTS=$(find . -type d \( -path ./node_modules -o -path ./vendor -o -path ./ios/Pods -o -path ./.husky \) -prune -o -name '*.sh' -print)
+SHELL_SCRIPTS=$(find . -type d \( -path "$(join_by_string " -o -path " "${DIRECTORIES_TO_IGNORE[@]}")" \) -prune -o -name '*.sh' -print)
 info "👀 Linting the following shell scripts using ShellCheck: $SHELL_SCRIPTS"
 info
 

--- a/scripts/shellUtils.sh
+++ b/scripts/shellUtils.sh
@@ -30,3 +30,12 @@ function assert_equal {
     success "Assertion passed: $1 is equal to $1"
   fi
 }
+
+# Usage: join_by_string <delimiter> ...strings
+function join_by_string {
+  local separator="$1"
+  shift
+  local first="$1"
+  shift
+  printf "%s" "$first" "${@/#/$separator}"
+}

